osdep/ManagedRoute.cpp

```diff
@@ -457,70 +471,66 @@ bool ManagedRoute::sync()
 			return false;
 	}
 
-	// Find lowest metric system route that this route should override (if any)
-	InetAddress newSystemVia;
-	char newSystemDevice[128];
-	newSystemDevice[0] = (char)0;
-	int systemMetric = 9999999;
 	std::vector<_RTE> rtes(_getRTEs(_target,false));
+
+	bool hasRoute = false;
+	rtes = (_getRTEs(_target,false));
```

Comment:
This seems redundant with line 474 above unless I'm missing something? (Which is 100% plausible.)

Reply:
Thanks. That's an silly one.